### PR TITLE
MM 1911 Added support for .jpeg in addition to .jpg

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

File: model/file.go [hunk 1]
```diff
@@ -13,8 +13,8 @@ const (
 )
 
 var (
-	IMAGE_EXTENSIONS = [4]string{".jpg", ".gif", ".bmp", ".png"}
-	IMAGE_MIME_TYPES = map[string]string{".jpg": "image/jpeg", ".gif": "image/gif", ".bmp": "image/bmp", ".png": "image/png", ".tiff": "image/tiff"}
+	IMAGE_EXTENSIONS = [5]string{".jpg", ".jpeg", ".gif", ".bmp", ".png"}
+	IMAGE_MIME_TYPES = map[string]string{".jpg": "image/jpeg", ".jepg": "image/jpeg", ".gif": "image/gif", ".bmp": "image/bmp", ".png": "image/png", ".tiff": "image/tiff"}
```
Comment (rgarmsen2295) [inline model/file.go]: Should jepg be jpeg?

**Comment by nickago** [comment]
Good catch, @rgarmsen2295 ready for re-review

**Comment by rgarmsen2295** [comment]
Looks good, +1

**Comment by crspeller** [comment]
+1

**Comment by hmhealey** [comment]
Looks good. Ready for merging

**Pull request merged**